<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowModel</class>
 <widget class="QDialog" name="windowModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelA">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_LAD_L">
            <property name="text">
             <string>LAD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_LCX_L">
            <property name="text">
             <string>LCX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_RCA_L">
            <property name="text">
             <string>RCA</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelB">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_LAD_R">
            <property name="text">
             <string>LAD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_LCX_R">
            <property name="text">
             <string>LCX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio_RCA_R">
            <property name="text">
             <string>RCA</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnRun">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
